.cirrus.yml

```diff
@@ -1,3 +1,4 @@
+azure_credentials: ENCRYPTED[eba4d3d664f53c26bf65dc0efb75ee288599d29dd6ae92a293d5df5b370ed986537b29e53da5f95cdd7ddf8d5a9e4e0d]
```

Comment:
You probably want to move it to `build_windows_task`. You can't have two credentials on one level.

Reply:
Same error after moved it to `build_windows_task`.